notebooks/239-image-bind/239-image-bind-quantize-nncf.ipynb

```diff
@@ -0,0 +1,1570 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    def laion2B_preprocess_train(example, image_column="url", text_column="caption"):
</pre><p>name of preprocessing looks confused, can we rename it to laion_preprocess? (in description below you mention lion-400m, but function claims laion-2b and train suffix also may lead to confusion as we do not perform QAT, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>